Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/ScrollingViewIntegrationTest.java

```diff
@@ -121,6 +124,8 @@ public void testScrollingViewDrawTool() {
 		longpressOnPointAndCheckIfCanvasPointHasChangedInXAndY(topRight);
 	}
 
+	// Ignore since Paintroid 419, needs to be rewritten.
+	@Ignore
```

Comment:
You can write the line like this: @Ignore("since Paintroid 419, needs to be rewritten."), so the reasoning is in the ignore line and we also don't get another warning, that a test was ignored without reasoning.